src/help_parser/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "help_parser"
+version = "0.0.0"
```

Comment:
Version can match the other versions for the uutils and uucore.

Reply:
Yep, I changed it.